package.json

```diff
@@ -12,15 +14,42 @@
       "isLibrary": false,
       "outputFormat": "esmodule"
     },
-    "main": {
-      "source": "src/pillarbox.js"
+    "pillarbox-core": {
+      "source": "src/pillarbox.js",
+      "distDir": "./dist",
+      "outputFormat": "esmodule",
+      "includeNodeModules": false,
+      "isLibrary": true,
+      "optimize": true
+    },
+    "pillarbox": {
+      "source": "build.js",
+      "distDir": "./dist",
+      "includeNodeModules": false,
+      "isLibrary": true,
+      "outputFormat": "esmodule",
+      "optimize": true
+    },
+    "pillarbox-css": {
+      "source": "node_modules/video.js/dist/video-js.css",
+      "distDir": "./dist",
+      "isLibrary": false,
+      "optimize": true
     }
   },
+  "files": [
+    "dist/*",
+    "docs/*"
+  ],
   "directories": {
     "doc": "docs"
   },
   "scripts": {
-    "build": "parcel build --target main",
+    "prebuild": "rm -rf dist",
```

Comment:
There is a cross-platform alternative to this command : https://www.npmjs.com/package/rimraf#cli

```
npm install --save-dev rimraf
````

```json
"scripts": {
  "prebuild": "rimraf 'dist'",
  ...
}
```

Reply:
Nice catch!